civihr_employee_portal/templates/civihr-employee-portal-custom-reports.tpl.php

```diff
@@ -105,6 +105,9 @@ class="btn btn-lg btn-secondary-outline text-uppercase">
             </div>
         </footer>
     </section>
+    <div data-graph-section="summary">
```

Comment:
The number of columns is dynamic, so I think we have to create them in JS. I could also add handlebars or something here, but that would be an overkill.